public/orbs-public-beta-own-infra.md

```diff
@@ -0,0 +1,74 @@
+# Running Orbs v2 on your own infrastructure
+
+## Prerequisites
+
+* A Linux machine with Docker Swarm
+* Boyar v1.1.1 [binary download](https://s3.amazonaws.com/orbs-network-releases/infrastructure/boyar/boyar-v1.1.1.bin)
+* Connection to Ethereum node (`$ETHEREUM_ENDPOINT` url)
```

Comment:
there is a link to a page setting up your infura account in the beta instructions page